src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      { age && <p className="Person__age">{`I am ${age}`}</p> }
```

Comment:
I understand the temptation to make it in one line, but you want to keep your code maximally clear)

readable code > short code
```suggestion
      { age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```